### PR TITLE
Implemented select

### PR DESCRIPTION
**Pull request by patrick91**
I'm working on project right now where I needed a select, so, here it is.

Let me know if you want to change anything.

### CODE REVIEW

**Comment by patrick91** [comment]
@claus do you have any feedback on this PR?

**Comment by claus** [comment]
Dang, i completely missed this! Will look into it asap.

**Comment by patrick91** [comment]
friendly ping :D

File: src/components/DatSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import result from 'lodash.result';
+import isString from 'lodash.isstring';
+import React, { PropTypes } from 'react';
+
+class DatString extends React.Component {
```
Comment (rreusser) [inline src/components/DatSelect.jsx]: Should be `DatSelect`, not `DatString`.

**Comment by rohan-deshpande** [comment]
Added via #2

**Pull request closed without merging**